projects/ks89/angular-modal-gallery/src/lib/components/previews/previews.html

```diff
@@ -6,9 +6,9 @@
   <ng-container *ngIf="previewConfig?.visible">
     <a class="nav-left"
        [attr.aria-label]="accessibilityConfig?.previewScrollPrevAriaLabel"
-       [tabIndex]="previewConfig?.arrows && start > 0 ? 0 : -1" role="button"
```

Comment:
The condition starting to be slighlty long (and each of them used twice), I've moved the logic in displayLeftPreviewsArrow() and displayRightPreviewsArrow()